Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb

```diff
@@ -5,12 +5,24 @@
 # Add Workspaces
 custom_log 'custom_log' do msg 'Adding Workspaces' end
 
+# Logging Powreshell Module input parameters for troubleshooting
+custom_log 'custom_log' do msg "-ModuleLoadPath: #{node['powershell_module']['import_module']}" end
+custom_log 'custom_log' do msg "-RelativityInstanceName: #{node['windows']['new_computer_name']}" end
+custom_log 'custom_log' do msg "-RelativityAdminUserName: #{node['relativity']['admin']['login']}" end
+custom_log 'custom_log' do msg "-RelativityAdminPassword: #{node['relativity']['admin']['password']}" end
+custom_log 'custom_log' do msg "-SqlAdminUserName: #{node['sql']['user']['eddsdbo']['login']}" end
+custom_log 'custom_log' do msg "-SqlAdminPassword: #{node['sql']['user']['sa']['password']}" end
+custom_log 'custom_log' do msg "-WorkspaceName: #{node['sample_workspace_name']}" end
+custom_log 'custom_log' do msg "-EnableDataGrid: false" end
+custom_log 'custom_log' do msg "For Datagrid enabled Workspace: -WorkspaceName: #{node['sample_data_grid_workspace_name']}" end
+custom_log 'custom_log' do msg "For Datagrid enabled Workspace: -EnableDataGrid: true" end
+
 powershell_script 'add_workspaces' do
   code <<-EOH
     #{node['powershell_module']['import_module']}
-    New-Workspace -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -WorkspaceName '#{node['sample_workspace_name']}' -EnableDataGrid false
-    
-    New-Workspace -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -WorkspaceName '#{node['sample_data_grid_workspace_name']}' -EnableDataGrid true
+    New-Workspace -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -WorkspaceName '#{node['sample_workspace_name']}' -EnableDataGrid false -ErrorAction Stop

```

Comment:
@jake-martin One more thing I noticed was this PowerShell module didn't stop when it encountered an exception and failed down the line. So I added "-ErrorAction Stop" to force the stop when there is an error.
This is one thing I forgot to mention to you. 

- [ ] Can you please add "-ErrorAction Stop" for rest of the PS modules calls?

Reply:
@calimetikcura Done. I re-ran the script to ensure there were no syntax errors.

Reply:
Awesome! Good job @jake-martin for running the scripts for confirmation!